extensions/markdown-language-features/src/commands/openImage.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Command } from '../commandManager';
+import { MarkdownPreviewManager } from '../preview/previewManager';
+
+export class OpenImageCommand implements Command {
+	public readonly id = '_markdown.openImage';
+
+	public constructor(
+		private readonly _webviewManager: MarkdownPreviewManager,
+	) { }
+
+	public execute(args: { resource: string; imageSource: string }) {
+		const source = vscode.Uri.parse(args.resource);
+		const imageSourceUri = vscode.Uri.file(vscode.Uri.parse(args.imageSource).path);
+		vscode.commands.executeCommand('vscode.open', imageSourceUri, this._webviewManager.findPreview(source));
```

Comment:
`findPreview` returns a preview which wont' be understood by `vscode.open`. Instead you want to get the view column where the preview is showing. This isn't exposed today but can be accessed from the `WebviewPanel` stored on a `MarkdownPreview`.

Reply:
I am reusing `openDocumentLink`. I think It covers what I am trying to do